server/src/index.js

```diff
@@ -1,5 +1,9 @@
 import express from 'express';
+import 'dotenv/config';
 import cors from 'cors';
+import db from './config/database';
+
+db.connectDb();
 
 const app = express();
 const port = process.env.PORT || 3001;
```

Comment:
Create a constant file for all process.envs. there you  can also set the defaults

Reply:
Done.